docs/articles/documentation/extending-testcafe/custom-reporter/creating-reporter-plugin/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: docs
+title: Creating Custom Reporter
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/
+---
+# Creating Reporter Plugin
+
+## Generating Reporter Project
+
+First, install [Yeoman](http://yeoman.io) and `generator-testcafe-reporter` using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+npm install -g generator-testcafe-reporter
+```
+
+Create a new directory where the generator should place your scaffolded project files and go into it.
+
+```bash
+mkdir MyReporter
+cd MyReporter
```

Comment:
As I mentioned before we should name dir as our project, and testcafe reporter should have name that starts with `testcafe-reporter-{your_suffix}`